src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -1,30 +1,31 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {WAD_INT} from "../MathLib.sol";
-
 /// @title ConstantsLib
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 library ConstantsLib {
-    /// @notice 1 bps = 0.01% (scaled by WAD).
-    int256 public constant ONE_BPS = 0.0001 ether;
```

Comment:
I still believe this was a good thing to have but anyway all of this is pinaillage